python_functions.py

```diff
@@ -0,0 +1,50 @@
+#Exercise_1
+def divide(x, y):
+    z = x / y
+    return z
+print(divide(6,2))
+
+numbers = [int(item) for item in input("Napiste cisla:").split()]
+print(numbers)
+def sum(list_of_numbers):

```

Comment:
Pozor, v editoru by to i melo byt videt, ale `sum` je prikaz z Pythonu a tedy jste jej nyni predefinovala.